setup.py

```diff
@@ -1,5 +1,9 @@
 import setuptools
-from velociraptor import __version__
+
+with open("velociraptor/__version__.py", "r") as v_file:
```

Comment:
What is the error that you're getting that requires this change?

Reply:
When you install velociraptor on a blank environment, importing it in the ``setup.py`` file triggers an error:  it automatically runs the ``__init__.py`` file, which requires ``unyt`` (and few other things) which are not available yet, and triggers the following error: 

```
          import unyt
      ModuleNotFoundError: No module named 'unyt'
      [end of output]
```

In the proposed way, instead of importing the variable from the file, you just read it and parse it.

Reply:
I'm not reproducing this error when I do a clean install, how are you doing your install? I think you should be installing the dependencies before installing the package. The current method is the standard way of doing it, so I don't really want to change it.

Reply:
Hello Rob! (@robjmcgibbon) Sorry for the delay in replying. If you start a new virtual env. and install velociraptor-python from the GitHub repository you find this error, I'm surprised this has not been flagged before. 

Please try the following:
```
python3 -m venv test_env
source test_env/bin/activate
git clone https://github.com/correac/velociraptor-python.git
cd velociraptor-python
python setup.py install
```

and you will enter into the error.

If instead you do:
```
git checkout -b Register_Nitrogen origin/Register_Nitrogen
python setup.py install
```

The installation should work.

Reply:
If I follow your steps then I get the same error. When I run `pip install -r requirements.txt` before `python setup.py install` I don't get an error, as that command will automatically install all the needed dependencies. I'd prefer to undo this change unless you still have issues even after installing the dependencies.

Reply:
You are right, if we first do pip install -r requirements it works. I have reverted the changes for setup.py